### ISSUE TITLE
Search Freezes after some time.

### ISSUE THREAD

**Issue by ThePlayingGoblin** [Zodsmar/SeedSearcherStandaloneTool#22]
So I've been searching for Mushroom Biome seeds and after interval of 5-10 minutes, the program stops searching for seeds, but still eats up cpu usage in the task manager. I was searching for 1.15.2 seeds and was using the exe version of the program. I will try again with 1.7.10 and the jar version.

**Comment by Zodsmar** [comment]
Can you explain more how the tool stop searching for seeds? Does the rejected seed count just stop counting up?

**Comment by Victorsueca** [comment]
I think I'm having the same issue.
Upon finding the first valid seed, the search halts, with this being the last console output:
```Valid Seed: 7630160103433182022
Exception in thread "Thread-4" java.lang.IllegalStateException: cannot open system clipboard
        at sun.awt.windows.WClipboard.openClipboard(Native Method)
        at sun.awt.windows.WClipboard.setContentsNative(Unknown Source)
        at sun.awt.datatransfer.SunClipboard.setContents(Unknown Source)
        at sassa.util.Util.console(Util.java:67)
        at sassa.main.BiomeSearcher.updateAcceptedWorldsProgress(BiomeSearcher.java:320)
        at sassa.main.BiomeSearcher.search(BiomeSearcher.java:371)
        at sassa.main.BiomeSearcher.run(BiomeSearcher.java:391)
        at java.lang.Thread.run(Unknown Source)```

After this, the tool will fail to output the found seed to the GUI and stop searching for further seeds.

**Comment by Zodsmar** [comment]
I was testing a way to have seeds clickable to copy to clipboard. I guess somehow something is getting through and causing the hiccup. I will look into it. Thanks for the info!

**Comment by Zodsmar** [comment]
Will be fixed in the next update v0.5.0
